src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -202,6 +205,46 @@ export class RoslynLanguageServer {
                 workspace: {
                     configuration: (params) => readConfigurations(params),
                 },
+                resolveCodeAction: async (codeAction, token, next) => {
```

Comment:
move this implementation to a separate file (can just export a function and call it from here